control-plane/api-gateway/cache/consul.go

```diff
@@ -59,7 +59,7 @@ const (
 	apiTimeout        = 5 * time.Minute
 )
 
-var Kinds = []string{api.APIGateway, api.HTTPRoute, api.TCPRoute, api.InlineCertificate, api.JWTProvider}
```

Comment:
are we just getting rid of inline certs?

Reply:
Yes, by default for consul-k8s we will be doing file-system-certificates as opposed to inline-certificates